jobbergate-agent/jobbergate_agent/settings.py

```diff
@@ -42,6 +42,7 @@ class Settings(BaseSettings):
     # Task settings
     TASK_JOBS_INTERVAL_SECONDS: int = Field(60, ge=10, le=3600)  # seconds
     TASK_GARBAGE_COLLECTION_HOUR: Optional[int] = Field(None, ge=0, le=23)  # hour of day
+    TASK_SELF_UPDATE_INTERVAL_SECONDS: int = Field(60 * 60, ge=10)  # seconds
```

Comment:
I think that making it optional and `None` by default makes more sense. So by default system admins are sure the version installed by them is pinned. I see use cases for auto update as well, but this should be explicitly set.

Reply:
check [8f30d49](https://github.com/omnivector-solutions/jobbergate/pull/518/commits/8f30d49012ff3e40f595c30247c90225547dd421)